test/cutting/test_cutting_evaluation.py

```diff
@@ -67,7 +66,10 @@ def test_execute_experiments(self):
             quasi_dists, coefficients = execute_experiments(
```

Comment:
These "quasi_dists" vars could be renamed to "results" throughout, but I'm trying to keep the diff clean for this PR

Reply:
`quasi_dists` is a bit nice, though, in the sense that it's less vague than `results`, which might e.g. be a `JobResult` or something if you only pay attention to the name.  (Up to you, though.)